lib/node-detector.js

```diff
@@ -36,7 +36,8 @@ class NodeDetector {
       log.debug(err);
       return null;
     }
-    let nodePath = stdout.replace(/[\n\r]/g, "");
+    // stdout can be multiple lines in windows environment. Read https://github.com/appium/appium-doctor/issues/33
+    let nodePath = stdout.split(/\r?\n/)[0];
```

Comment:
There is os.EOL constant for this purpose

Reply:
this will throw an exception if stdout is empty

Reply:
https://nodejs.org/api/os.html#os_os_eol oh, it's helpful